From cd465e01afe9808f29d16eed00ab01d8f4e60f4a Mon Sep 17 00:00:00 2001
From: port-worker <port-worker@local>
Date: Tue, 11 Aug 2026 08:43:57 +0000
Subject: [PATCH 07/27] Revert "Set /system/xbin permissions to 750."

This reverts commit 42a1a126e554a8bca31d0afc832848b7b0fa1f4e.

Change-Id: I277def5153616d6f00bc877e6170b7fed7cd799d
[forward-ported to A16: same fs_config.cpp entry, shifted line]

Change-Id: I1561d51ef61dfb55c28d5bfd700f78c4732e9608
---
 libcutils/fs_config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcutils/fs_config.cpp b/libcutils/fs_config.cpp
index f28c897ed..01909816a 100644
--- a/libcutils/fs_config.cpp
+++ b/libcutils/fs_config.cpp
@@ -77,7 +77,7 @@ static const struct fs_path_config android_dirs[] = {
     { 00751, AID_ROOT,         AID_SHELL,        0, "system/bin" },
     { 00755, AID_ROOT,         AID_ROOT,         0, "system/etc/ppp" },
     { 00755, AID_ROOT,         AID_SHELL,        0, "system/vendor" },
-    { 00750, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
+    { 00751, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system_ext/bin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "vendor/bin" },
     { 00755, AID_ROOT,         AID_SHELL,        0, "vendor" },
-- 
2.43.0

